src/neo/SmartContract/Native/Votes/Method/FindAllPathMethod.cs

```diff
@@ -0,0 +1,207 @@
+using System;
+using System.Collections.Generic;
+
+namespace Neo.SmartContract.Native
+{
+    public class FindAllPathMethod
+    {
+        /// <summary>
+        /// Node Set
+        /// </summary>
+        private List<int> nodesList = new List<int>();
+        /// <summary>
+        /// Vector Graphic,describe the connection between nodes and nodes
+        /// </summary>
+        private Dictionary<String, int> vectorGraphic = new Dictionary<String, int>();
+        /// <summary>
+        /// Edge Array,describe the connection between nodes and nodes
+        /// </summary>
+        private int[,] nodesEdgesMatrix;
+        /// <summary>
+        /// Path Set,store paths in vector graphic.
+        /// </summary>
+        private List<List<int>> path = new List<List<int>>();
+
+        public FindAllPathMethod(List<int> nodesList, Dictionary<String, int> vectorGraphic)
+        {
+            this.nodesList = nodesList;
+            this.vectorGraphic = vectorGraphic;
+
+            CreateGraphMatrix();
+        }
+
+        /// <summary>
+        /// create edge array
+        /// </summary>
+        private void CreateGraphMatrix()
+        {
+            int nodeCount = nodesList.Count;
+            nodesEdgesMatrix = new int[nodeCount, nodeCount];
+            for (int i = 0; i < nodeCount; i++)
+                for (int j = 0; j < nodeCount; j++)
+                    nodesEdgesMatrix[i, j] = -1;
+            foreach (KeyValuePair<String, int> entry in vectorGraphic)
+            {
+                String[] temp = entry.Key.Split(new char[] { ',' });
+                String startNode = temp[0];
+                int startNodeIndex = System.Convert.ToInt32(temp[0]);
+                String endNode = temp[1];
+                int endNodeIndex = System.Convert.ToInt32(temp[1]);
+                int weight = entry.Value;
+                nodesEdgesMatrix[startNodeIndex, endNodeIndex] = weight;
+            }
+        }
+
+        /// <summary>
+        /// Find all paths from startNode to endNode
+        /// </summary>
+        /// <param name="startNode">startNode</param>
+        /// <param name="endNode">endNode</param>
+        /// <returns>path set</returns>
+        public List<List<int>> FindPath(int startNode, int endNode)
+        {
+            path.Clear();
+            List<int> tempUnUsedNodes = new List<int>();
+            List<int> tempPath = new List<int>();
+            tempUnUsedNodes.AddRange(nodesList);
+            tempUnUsedNodes.Remove(startNode);
+            CreatePath(tempUnUsedNodes, startNode, endNode, tempPath);
```

Comment:
plz, don't use `temp` to name variables. 

`tempUnUsedNodes` -> `unUsedNodes`
`tempPath` -> `emptyCreatedPath`